lib/graphiti/active_graph/sideload_resolve.rb

```diff
@@ -1,5 +1,19 @@
 module Graphiti::ActiveGraph
   module SideloadResolve
+    def initialize(object, resource, query, opts = {})
+      @object = object
+      @resource = resource
+      @query = query
+      @opts = opts
+
+      opts[:query_obj] = @query
+      @unpaginated_object = opts[:unpaginated_query].presence || @object
```

Comment:
When I see the original method [here](https://github.com/graphiti-api/graphiti/blob/master/lib/graphiti/scope.rb#L5), I don't see these two lines. why do we need them?

Reply:
We use `unpaginated_object` for stats as well. So even if we skip filtering we need it (resource_proxy.rb:46). Not sure about `opts[:query_obj]`. I copied it here so that these 2 lines are in the same place.

Reply:
ok `unpaginated_object` we can keep considering it is part of `graphiti-activegraph`'s code, but I don't see any mention of `opts[:query_obj]` in `graphiti-activegraph`. So unless it is required anywhere we should not have it here.

Reply:
moved it to api